update_box86.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd /home/pi/box86
```

Comment:
Maybe use instead
```suggestion
cd "$(dirname "$0")"
```
so it is more portable? That would imply that there is no need to `cp` into `box86/backup` later, only to `backup`.